typescript/packages/subsurface-viewer/src/layers/map/mapLayer.stories.tsx

```diff
@@ -871,6 +873,91 @@ NodeCenteredPropMapWithArrayInput.parameters = {
     },
 };
 
+function makeGaussian(amplitude, x0, y0, stdX, stdY) {
+    return function (amplitude, x0, y0, stdX, stdY, x, y) {
+        const exponent = -(
+            Math.pow(x - x0, 2) / (2 * Math.pow(stdX, 2)) +
+            Math.pow(y - y0, 2) / (2 * Math.pow(stdY, 2))
+        );
+        return amplitude * Math.pow(Math.E, exponent);
+    }.bind(null, amplitude, x0, y0, stdX, stdY);
+}
+
+function makeData(n: number, amplitude: number): Float32Array {
+    const t0 = performance.now();
+    const X0 = 0;
+    const Y0 = 0;
+    const stdX = 75;
+    const stdY = 50;
+    const f = makeGaussian(amplitude, X0, Y0, stdX, stdY);
+
+    const data = new Float32Array(n * n).map((val, index) => {
+        const x = (index % n) - n / 2;
+        const y = Math.floor(index / n) - n / 2;
+        return f(x, y); // keep + 0.3 * Math.random();
+    });
+
+    const t1 = performance.now();
+    console.log(`makeData took ${(t1 - t0) * 0.001}  seconds.`);
+    return data;
+}
+
+//-- NodeCenteredPropMap  with native javascript arrays as input --
+// Mesh will be n x n nodes.
+const n = 300;
```

Comment:
Would be nice to expose these variables as arguments to the story. There are some examples in some other stories. I can give some pointers.

Reply:
Done